aurora-postgresql-plugin/pom.xml

```diff
@@ -98,12 +98,12 @@
         <configuration>
           <instructions>
             <_exportcontents>
-              io.cdap.plugin.auroradb.postgres.*;
-              io.cdap.plugin.db.batch.source.*;
-              io.cdap.plugin.db.batch.sink.*;
-              org.apache.commons.lang;
-              org.apache.commons.logging.*;
-              org.codehaus.jackson.*
+                io.cdap.plugin.auroradb.postgres.*;
+                io.cdap.plugin.db.source.*;
+                io.cdap.plugin.db.sink.*;
+                org.apache.commons.lang;
```

Comment:
similar comments everywhere. With these exports it seems like things would break if we used multiple of them in the same pipeline.